pkg/kubelet/metrics/metrics.go

```diff
@@ -168,6 +173,12 @@ var (
 		{60 * 1024 * 1024 * 1024, "60GB-100GB"},
 		{100 * 1024 * 1024 * 1024, "GT100GB"},
 	}
+	// DRADurationBuckets is the bucket boundaries for DRA operation duration metrics
+	// DRAOperationsDuration and DRAGRPCOperationsDuration defined below in this file.
+	// The buckets max value 50 is based on the 45sec max gRPC timeout value defined
+	// for the DRA gRPC calls in the pkg/kubelet/cm/dra/plugin/registration.go
+	// Additional 5 seconds are used to cover an overhead added by Kubelet operations.
+	DRADurationBuckets = metrics.ExponentialBucketsRange(.1, 50, 20)
```

Comment:
It looks odd that this is not monotonically increasing. Are you sure that this works? Speaking of "it works", what about tests for this?

It also seems a bit coarse to me and I am not sure whether "buckets max value 50" makes sense: doesn't that create a bucket for "times >= 50", which will almost certainly always be empty?

Reply:
> It looks odd that this is not monotonically increasing.

Never mind. This isn't a list of values as in the previous default.

20 buckets seems a bit much to me.

The comment about 50 as max still applies.

Reply:
> doesn't that create a bucket for "times >= 50", which will almost certainly always be empty?

metrics.ExponentialBucketsRange(.1, 50, 20) returns `[]float64 len: 20, cap: 20, [0.1,0.1386918856530028,0.192354391459856,0.26677993265203287,0.37000211913891523,0.5131629159898314,0.7117153246582317,0.9870914042498925,1.36901568167288,1.8987136637974276,2.6333617834718677,3.652259113562678,5.065387033533722, 7.02528079243062, 9.743494403440243, 9.743494403440243, 13.513436116626067, 18.742039366662606, 25.993687807452467, 36.051135770910506, 49.999999999999986`. I don't think there is a place for `times >= 50`. May be I simply misunderstood what do you mean. Can you elaborate?

Reply:
> 20 buckets seems a bit much to me.

Decreased to 15:
```
[]float64 len: 15, cap: 15, [0.1,0.1558775502054779,0.2429781065806128,0.3787483200735143,0.5903836027749962,0.9202754968205039,1.434502899585092,2.2360679774997876,3.4855279842558455,5.433155633584387,8.469069900482252,13.20137868606124,20.577985689180366,32.07645997392817,49.999999999999915}
```

Reply:
I think that last `49.999999999999986` creates a bucket `[49.999999999999986, inf]` which will always be empty. We could make the last bucket `[40, inf]` and avoid one useless bucket?

Or do I misunderstand what the histogram collects as data?